lib/bike_brigade_web/live/campaign_signup_live/index.html.heex

```diff
@@ -1,8 +1,9 @@
 <nav
+  :if={@showing_urgent_campaigns == false}
```

Comment:
I think having an easy way to navigate from this page to all delivery signups without going back to the hamburger menu would be good!